src/bindings/python/flux/job/list.py

```diff
@@ -97,6 +97,44 @@ def job_list_id(flux_handle, jobid, attrs=[]):
     return rpc
 
 
+# get_job is the single variant of job_list_id, and returns the
+# expected data structure (dict) to describe one job (or None)
+def get_job(flux_handle, jobid):
+    """Get job information dictionary based on a jobid
+
+    This is a courtesy function for the user that looks for it
+    in the flux.job namespace after submitting a job and wanting
+    information about it.
```

Comment:
I realize this is a lot of stuff to write, but it would probably be good to describe the contents of the dictionary. Especially since the docstring will be pulled into the docs